instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -135,6 +136,20 @@ func WithMeterProvider(mp metric.MeterProvider) Option {
 	return meterProviderOption{mp: mp}
 }
 
+type spanStartOption struct{ opts []trace.SpanStartOption }
+
+func (o spanStartOption) apply(c *config) {
+	if o.opts != nil {
```

Comment:
This `if` is not actually needed, right?

Everything else looks good, thanks!

Reply:
You are correct.